src/styles/_mixin.scss

```diff
@@ -0,0 +1,21 @@
+@mixin numberOfDay {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: $higth-box ;
+}
+
+@mixin boxCreate {
+  box-sizing: border-box;
+  width: $width-box;
+  height: $higth-box;
+  background-color: $color-grey;
+  border: 1px solid #000;
+  transition: transform 0.5s, background-color 0.5s;
+}
+
+@mixin moveBox {
+  cursor: pointer;
```

Comment:
`cursor: pointer` should go under general selector, not `:hover` rules. It is only seen on hover anyway.